workspaces/config/lib/definitions/definitions.js

```diff
@@ -1826,6 +1827,9 @@ define('save-exact', {
     version rather than using npm's default semver range operator.
   `,
   flatten (key, obj, flatOptions) {
+    if (obj[key]) {
+      flatOptions.save = true
```

Comment:
This is not the same as the other flags, it is not telling npm to save, it is telling it what prefix to use when it does save.  Folks override this a lot and it should not force a `save` state.

Reply:
That makes sense. I removed the changes to save-exact.